spec/parser_spec.rb

```diff
@@ -55,4 +55,18 @@
     end
   end
 
+  Dir.glob('spec/support/edi_files/valid/856/18.EDI').each do |file|
+    context file do
+      it 'should include error when set856 :id => "PID" is "Pid"' do
+        input = File.read(file)
+        # Set PID to Pid.
+        Baldr::Grammar::Version4010::Set856.structure[:level][2][:level][5][:id] = 'Pid'
+        parser = Baldr::Parser.new(input)
+        parser.error.should include "segment SE is required, but nothing was found"
+        # Reset to prevent remaining tests from breaking.
+        Baldr::Grammar::Version4010::Set856.structure[:level][2][:level][5][:id] = 'PID'
```

Comment:
@npersad84 after you set it correctly, you need to check the expectation again such that it doesn't include the error. This only shows one-part of the problem, which is breaking it, not fixing it